### PR TITLE
CI aws git checkout

### PR DESCRIPTION
**Pull request by General-Beck**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -209,6 +196,7 @@ build-android:
   image:                           parity/rust-android:gitlab-ci
   variables:
     CARGO_TARGET:                  armv7-linux-androideabi
+  dependencies:
```
Comment (TriplEight) [inline .gitlab-ci.yml]: This line prevents artifacts from other jobs to be downloaded before this job.
It makes sense to do the same for the other build jobs.

Comment (TriplEight) [reply .gitlab-ci.yml]: later

Comment (gabreal) [reply .gitlab-ci.yml]: would suggest to use
```
dependencies: []
```
to make it consistent with the `cache: {}` statement.

File: scripts/gitlab/publish-aws.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e # fail on any error
+set -u # treat unset variables as error
+
+echo "__________Push binaries to AWS S3____________"
+case "${SCHEDULE_TAG:-${CI_COMMIT_REF_NAME}}" in
+    (beta|stable|nightly)
+      export BUCKET=releases.parity.io/ethereum;
```
Comment (gabreal) [inline scripts/gitlab/publish-aws.sh]: no need to export that variable, it is only used within this script.

File: .gitlab-ci.yml (hunk 1 above)
Comment (TriplEight) [reply .gitlab-ci.yml]: indeed

**Pull request merged**